lldb/test/Shell/SymbolFile/DWARF/no_unique_address-alignment.cpp

```diff
@@ -0,0 +1,25 @@
+// XFAIL: *
+
+// RUN: %clangxx_host -gdwarf -o %t %s
+// RUN: %lldb %t \
+// RUN:   -o "b main" \
```

Comment:
It looks like you're not actually running the binary. Do you need the breakpoint?

Reply:
good catch, removed